app/controllers/projects_controller.rb

```diff
@@ -70,12 +70,21 @@ def update
     update! do |format|
       format.html do
         if resource.errors.present?
-          flash[:alert] = resource.errors.full_messages.to_sentence
+          error_message = ''
+          resource.errors.each do |error|
+            error_message += resource.errors[error][0]
+            error_message += '<br/>'
+          end
+          flash[:alert] = error_message.html_safe
         else
           flash[:notice] = t('project.update.success')
         end
 
-        redirect_to edit_project_path(@project, anchor: 'basics')
+        if params[:anchor]
```

Comment:
I believe that the anchor will not be in params, besides, we should have a spec that cover the first block of the if. Probably only the else is being executed.